siatest/renter/renter_test.go

```diff
@@ -75,3 +79,106 @@ func testUploadDownload(t *testing.T, tg *siatest.TestGroup) {
 		t.Error(err)
 	}
 }
+
+// testRenterLocalRepair tests if a renter correctly repairs a file from disk
+// after a host goes offline.
+func testRenterLocalRepair(t *testing.T, tg *siatest.TestGroup) {
+	// Grab the first of the group's renters
+	renter := tg.Renters()[0]
```

Comment:
need a check both here and in the other test that there are at least 2 hosts, otherwise this is going to fail.